Source/Core/Core/IOS/SDIO/SDIOSlot0.h

```diff
@@ -111,6 +112,15 @@ class SDIOSlot0 : public Device
     EVENT_INVALID = 0xc210000
   };
 
+  enum class SDProtocol
+  {
+    V1 = 0,
```

Comment:
Since the values are not actually being used, you don't need to have = 0 and = 1